docs/tutorials/web.md

```diff
@@ -5,10 +5,405 @@ has_children: false
 nav_order: 7
 ---
 
-# Deploy ML models on the web
+# Deploy and inference ML models on the web
 
- Check out the [release blog here](https://cloudblogs.microsoft.com/opensource/2021/09/02/onnx-runtime-web-running-your-machine-learning-model-in-browser/)
+One of the hardest parts when deploying and inferencing in languages that are not commonly used for data science is figuring out how to do the data processing and inferencing. We have done all the hard work for you with [this template](https://github.com/microsoft/onnxruntime-nextjs-template)! In this tutorial we will use a template QuickStart to start building out a static web app using ORT-web. We will do the inference in JavaScript on the browser for a computer vision model. Let's learn a bit more about the library, ONNX Runtime Web, which allows us to inference in many different languages.
 
+Example template output:
```

Comment:
Can we describe what the application does in a little more detail?

Reply:
added